app/components/message/message.css

```diff
@@ -11,3 +11,19 @@
   color: var(--color-text-light);
   font-size: var(--font-size-s);
 }
+
+.Message-photos {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding-bottom: var(--spacing-unit);
+}
+
+.Message-photos > img {
+  border: 3px solid rgba(0, 0, 0, 0.05);
+  max-width: 100%;
+  height: auto;
+}
+.Message-photos > img + img {
+  margin-top: var(--spacing-unit);
+}
```

Comment:
You *could* just wrap the images in a `stack` – no need for an additional rule set then.